ui/apps/platform/cypress/helpers/request.js

```diff
@@ -114,3 +114,13 @@ export function interactAndInspectGraphQLVariables(interactionCallback, opname)
         return cy.wrap(interception.request.body.variables);
     });
 }
+
+export function expectRequestedSort(expectedSort) {
+    return (variables) => {
+        const { sortOption } = variables.pagination;
+        expect(sortOption).to.deep.equal(
+            expectedSort,
+            `Expected sort option ${JSON.stringify(expectedSort)} but received ${JSON.stringify(sortOption)}`
+        );
+    };
+}
```

Comment:
Moving this to `requests.js` as it is more related to testing a property of the request, than testing sorting specifically. This pattern will follow in future PRs for filtering and pagination.